build.gradle

```diff
@@ -62,8 +63,9 @@ gradlePlugin {
 
 bintray {
   publish = true
-  user = project.hasProperty('bintrayUser') ? project.property('bintrayUser') : System.getenv('BINTRAY_USER')
-  key = project.hasProperty('bintrayApiKey') ? project.property('bintrayApiKey') : System.getenv('BINTRAY_API_KEY')
+  user = findProperty("BINTRAY_USER") ?: "no.bintray.user"
```

Comment:
Looks like `findProperty()` just checks project gradle properties and returns null if they're not found. Is it intended to not fallback to environment variables anymore? The fallback is a nice to have imho, since it opens the door to either declaring them as both gradle properties (both here or in the home `.gradle/gradle.properties` file) and system variables.

Reply:
`project.properties[key]` and`System.getenv(key)`, `findProperty` search on both sides

Reply:
[Does it](https://docs.gradle.org/current/javadoc/org/gradle/api/Project.html#findProperty-java.lang.String-)?

Reply:
Oh it was using the gradle one, my bad